src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -402,10 +403,39 @@ private static FormulaValue DateParse(EvalVisitor runner, EvalVisitorContext con
             }
         }
 
+        private static bool TryGetCulture(string name, out CultureInfo value)
```

Comment:
>TryGetCulture [](http://example.com/codeflow?start=28&length=13)

FWIW - I was skeptical about the foreach here, but  according to SO, that's the best way to do! 
[https://stackoverflow.com/questions/13723790/how-to-check-exists-culture-in-net](https://stackoverflow.com/questions/13723790/how-to-check-exists-culture-in-net)